src/random.cpp

```diff
@@ -564,9 +564,11 @@ void RandAddPeriodic() noexcept { ProcRand(nullptr, 0, RNGLevel::PERIODIC); }
 void RandAddEvent(const uint32_t event_info) noexcept { GetRNGState().AddEvent(event_info); }
 
 bool g_mock_deterministic_tests{false};
```

Comment:
Setting `g_mock_deterministic_tests` is not a great alternative as it makes `GetRand` always return the same value.